rust/processor/src/worker.rs

```diff
@@ -356,6 +356,19 @@ impl Worker {
                 "[Parser] Successfully fetched transaction batches from channel."
             );
 
+            let last_transaction_pb_timestamp = transactions_batches
+                .last()
+                .unwrap()
+                .transactions
+                .as_slice()
+                .last()
+                .unwrap()
+                .timestamp
+                .clone();
+            let last_transaction_timestamp = last_transaction_pb_timestamp.map(|t| {
+                chrono::NaiveDateTime::from_timestamp_opt(t.seconds, t.nanos as u32).unwrap()
```

Comment:
Use `timestamp_to_unixtime`

Reply:
Yes please!

Reply:
done.

Reply:
just realized that the destination of this variable is db, and the expected type is TIMESTAMP. If we did it here, we might need another conversion back to NativeDate type.

Reply:
i guess it depends on how it's gonna be used. all the prom metrics use unixtime, so if we're comparing against prom metrics it makes more sense to use unixtime. maybe 2 columns - one for human readable timestamp and one for unixtime

Reply:
Please use this https://github.com/aptos-labs/aptos-indexer-processors/blob/main/rust/processor/src/utils/util.rs#L234-L245

Reply:
good call. updated.